Bottom: 8e0dc746b8da6e0e7eccfc8abbe58b9980c43166
Top:    7fe099620f5b46a36f5bf97d3524579c8c7ec2bb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-08 02:06:33 +0000

Avoid clearing pins / vias from polygons... it is already done!






---

diff --git a/src/draw.c b/src/draw.c
index 545c6e9..dd4ff4d 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1262,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1284,6 +1285,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
